tests/torch/test_sota_checkpoints.py

```diff
@@ -557,8 +573,9 @@ def test_train(self, cuda_ip, sota_data_dir, sota_checkpoints_dir, eval_test_str
 @pytest.fixture(autouse=True, scope="class")
 def openvino_preinstall(ov_data_dir):
     if ov_data_dir:
-        subprocess.run("pip install -r requirements_onnx.txt", cwd=MO_DIR, check=True, shell=True)
-        subprocess.run(f"{sys.executable} setup.py install", cwd=ACC_CHECK_DIR, check=True, shell=True)
+        if 'deployment_tools' in MO_DIR.parts and 'deployment_tools' in ACC_CHECK_DIR.parts:
```

Comment:
Set up a global variable or a fixture instead of checking for this every time, and call this global variable appropriately.